backend/entityservice/tests/conftest.py

```diff
@@ -71,44 +92,58 @@ def create_project_response(requests, size, overlap, result_type, encoding_size=
         "dp_2": <JSON RESPONSE TO DATA UPLOAD>
     }
     """
-    project, (dp_1, dp_2) = create_project_upload_fake_data(
+    project, dp_responses = create_project_upload_fake_data(
         requests, size, overlap=overlap, result_type=result_type, encoding_size=encoding_size)
     project.update({
         'size': size,
         'encoding-size': encoding_size,
         'upload-mode': 'JSON',
         'overlap': overlap,
-        'dp_1': dp_1,
-        'dp_2': dp_2
+        'dp_responses': dp_responses
     })
     return project
 
 
-@pytest.fixture(scope='function', params=PROJECT_PARAMS)
+@pytest.fixture(scope='function', params=PROJECT_PARAMS_2P)
 def mapping_project(request, requests):
     size, overlap, encoding_size = request.param
     prj = create_project_response(requests, size, overlap, 'mapping', encoding_size)
     yield prj
     delete_project(requests, prj)
 
 
-@pytest.fixture(scope='function', params=PROJECT_PARAMS)
+@pytest.fixture(scope='function', params=PROJECT_PARAMS_2P)
 def similarity_scores_project(request, requests):
     size, overlap, encoding_size = request.param
     prj = create_project_response(requests, size, overlap, 'similarity_scores', encoding_size)
     yield prj
     delete_project(requests, prj)
 
-
-@pytest.fixture(scope='function', params=PROJECT_RESULT_TYPES)
-def result_type(request):
+@pytest.fixture(scope='function', params=tuple(itertools.chain(
+    [(t, 2) for t in PROJECT_RESULT_TYPES_2P],
+    [(t, n) for t in PROJECT_RESULT_TYPES_NP for n in NUMBERS_PARTIES])))
+def result_type_number_parties(request):
     yield request.param
 
 
-@pytest.fixture(scope='function', params=PROJECT_RESULT_TYPES)
-def project(request, requests):
-    result_type = request.param
-    project = create_project_no_data(requests, result_type)
+@pytest.fixture(scope='function')
+def result_type(request, result_type_number_parties):
+    result_type, _ = result_type_number_parties
+    yield result_type
+
+
+@pytest.fixture(scope='function')
+def number_parties(request, result_type_number_parties):
+    _, number_parties = result_type_number_parties
+    yield number_parties
```

Comment:
`result_type` and `number_parties` are tied through `result_type_number_parties` to always be compatible.